packages/myst-to-typst/src/table.ts

```diff
@@ -34,7 +34,9 @@ export const tableHandler: Handler = (node, state) => {
     return;
   }
   state.useMacro('#import "@preview/tablex:0.0.7": tablex, cellx');
-  state.write(`${command}(columns: ${columns}, header-rows: ${countHeaderRows(node)},\n`);
+  state.write(
+    `${command}(columns: ${columns}, header-rows: ${countHeaderRows(node)}, repeat-header: true,\n`,
```

Comment:
`repeat-header` is documented in tablex, but it didn't seem to work super well. There is a note in the docs that behaviour may be somewhat erratic...